src/snac/client/webui/templates/edit_components.html

```diff
@@ -801,7 +801,7 @@ <h4 class="text-center" id="nameEntry_heading_{{i}}">{{nameEntry.original}}</h4>
 
 
 
-{% macro displayConstellationRelation(X, inEdit, showSubElements, relation, i, entityType, preferredName, reverse) %}
+{% macro displayConstellationRelation(X, inEdit, showSubElements, relation, i, entityType, preferredName, reverse, control) %}
```

Comment:
So if `control` wasn't being passed into displayConstellationRelations, were all those internal links generated with the wrong base URL?  I've clicked them locally and hadn't noticed. Good catch.

Do you have any thoughts on whether it'd be a good idea to just make the snacURL a twig global variable for scoping purposes?

Reply:
Haha, yeah, they were displayed without the correct base URL, so the redirected to /view/... and I must have missed it since I add the simplified URL structure.  It works fine on snaccooperative.org and snac-dev.iath.virginia.edu because an empty base url would work.

I think we could make both control and X global variables.